roles/openshift_cfme/files/miq-backup-pvc.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: manageiq-backup
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 15Gi
```

Comment:
Consider templatizing these variables in case the user wants to change sizes.

Reply:
same note as below, only `miq-pv-*` are being used right now. Those other ones were copied over from the upstream repo.